examples/chip-tool/commands/common/CHIPCommand.cpp

```diff
@@ -183,6 +150,40 @@ void CHIPCommand::MaybeTearDownStack()
     StopTracing();
 }
 
+CHIP_ERROR CHIPCommand::EnsureCommissionerForIdentity(std::string identity)
+{
+    if (mCommissioners.find(identity) != mCommissioners.end())
+    {
+        return CHIP_NO_ERROR;
+    }
+
+    // Need to initialize the commissioner.
+    chip::FabricId fabricId;
+    if (identity == kIdentityAlpha)
+    {
+        fabricId = kIdentityAlphaFabricId;
+    }
+    else if (identity == kIdentityBeta)
+    {
+        fabricId = kIdentityBetaFabricId;
+    }
+    else if (identity == kIdentityGamma)
+    {
+        fabricId = kIdentityGammaFabricId;
+    }
+    else
+    {
+        fabricId = strtoull(identity.c_str(), nullptr, 0);
+        if (fabricId < kIdentityOtherFabricId)
+        {
+            ChipLogError(chipTool, "Invalid identity: %s", identity.c_str());
+            return CHIP_ERROR_INVALID_ARGUMENT;
+        }
+    }
```

Comment:
So how do you create a FabricId 1234 under root of trust (identity) Alpha?

Reply:
You don't, either before or after this PR.  This PR is not changing the basic functionality, which is that "identity"means "fabric" in chip-tool.  It does not mean "root of trust".